dbms/src/Operators/Operator.h

```diff
@@ -0,0 +1,89 @@
+// Copyright 2023 PingCAP, Ltd.
```

Comment:
It is so luxury to NOT have an `Operator` class in an execution engine. I suggest creating a common base class, named `Operator`, and all the sub-classes having `Op` suffix.

Reply:
ok, updated.